VideoCallKitQuickStart/ViewController+CallKit.swift

```diff
@@ -66,7 +73,12 @@ extension ViewController : CXProviderDelegate {
          * Configure the audio session, but do not start call audio here, since it must be done once
          * the audio session has been activated by the system after having its priority elevated.
          */
-        TVIAudioController.shared().configureAudioSession(.videoChatSpeaker)
+
+        // Stop the audio unit by setting isEnabled to `false`.
+        self.audioDevice.isEnabled = false;
+
+        // Configure the AVAudioSession by executign the audio device's `block`.
+        self.audioDevice.block()
```

Comment:
Minor, you could move lines 72 to 81 into `performRoomConnect` since Starting and Answering a Call are the same W.R.T. audio.

Reply:
I would keep outside to show it explicitly on how to configure audio device. `performRoomConnect` has the _general sdk code_ to connect to a Room.

Reply:
Sounds fine to me, just something I noticed reading the code. I think you are correct that being explicit about the configuration steps is best (even if some duplication is done).